### DISCUSSION TITLE
Collapse / Expand Log/Event

### DISCUSSION THREAD

**Discussion by nikuscs** [spatie/ray#459] [Ideas]
Hey Spatie Team 👋

Would make sense to have an option to Log items starting with collapsed state instead of expanded ?
For instance we are using ray to debug some external HTTP requests, and indeed it takes a lot of screen space to check the logs, would be perfect if we could achieve something like :

```php
ray($request)->small()->collapsed()
ray($request)->small()->expand()
```

Example of what its wanted : 

![image](https://user-images.githubusercontent.com/12662462/119876814-32f97500-bf20-11eb-9b0c-5dbc27b81135.png)

Not sure if this is already present or not but based on my findings is not, in case it does just delete this.

**Comment by freekmurze** [comment]
Makes sense, we'll add this soon.

**Reply by AdrianMrn** [reply]
@igerslike @freekmurze I believe this is already an option (only collapsing):

`ray('hi')->hide();`

It seems like we documented this for other frameworks, but not for Laravel: https://spatie.be/docs/ray/v1/usage/framework-agnostic-php-project#hiding-items

**Reply by freekmurze** [reply]
Right 👍

**Reply by nikuscs** [reply]
indeed the hide does that, thanks a lot!  I always though **hide** would stand for hiding the window, but there is a hideApp() method as well, in any case collapsing/expanding also might be a thing. Thanks a lot @AdrianMrn for the catch up.
